### DISCUSSION TITLE
How to export all my data and projects from GitHub?

### DISCUSSION THREAD

**Discussion by egorsmkv** [community/community#23537] [New to GitHub]
<p>Hello.</p>
<p>I’d like to export all my projects (public and private) from GitHub. How can I do this?</p>
<p>I have found this tool - <a href="https://github.com/amitsaha/gitbackup" class="inline-onebox" rel="noopener nofollow ugc">GitHub - amitsaha/gitbackup: Tool to backup your Bitbucket, GitHub and GitLab repositories</a> but I’d like to hear from others.</p>
<p>Thanks.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>is this what you are looking for?</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077789-1e693447-ab3d-44cb-be03-6f21313ca501.png" alt="image" data-base62-sha1="wdIjmiiNBDYGwttWzfnvdeRjY83" width="649" height="164"></p>
<p>it’s at the Settings,</p>
<p><a href="https://github.com/settings/admin">https://github.com/settings/admin</a></p>

**Comment by egorsmkv** [comment]
<p>Yes. It is what I was looking for.</p>
<p>Thanks.</p>
